static/bidder-params/kargo.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Kargo Adapter Params",
+    "description": "A schema which validates params accepted by the Kargo adapter",
+    "type": "object",
+  
+    "properties": {
+        "tagid": {
+          "type": "string",
+          "description": "An ID which identifies the adslot placement. Equivalent to the id of target inventory, ad unit code, or placement id"
+        }
+      },
+    
+    "required": ["tagid"]
```

Comment:
I believe prebid sets this field as required, so it either needs to specify a parameter or have no parameters. If adSlot is optional, then this field should be left empty since Prebid Server Core will discard them.

Reply:
Updated with `adSlotID`